### PR TITLE
[QA-848] feat: Get methods for data streaming API

### PR DESCRIPTION
**Pull request by JoiceFelix**
### Jira Issues:
[QA-881 - [openAPI] List all data streamings](https://aziontech.atlassian.net/browse/QA-881)
[QA-889- [openAPI] List all domains to use on data streaming](https://aziontech.atlassian.net/browse/QA-889)
[QA-890- [openAPI] Retrieve data streaming by ID](https://aziontech.atlassian.net/browse/QA-890)
[QA-891- [openAPI] List data streaming's templates](https://aziontech.atlassian.net/browse/QA-891)
[QA-892- [openAPI] Retrieve data streaming's template by ID](https://aziontech.atlassian.net/browse/QA-892)

### CODE REVIEW

File: data_streaming.yml [hunk 1]
```diff
@@ -28,14 +27,152 @@ paths:
...
+        '400':
+          description: Bad Request
+        '401':
+          description: Unauthorized
+        '403':
+          description: Forbidden
+        '404':
+          description: Not Found
+        '429':
+          description: Rate Limit
```
Comment (rodrigomaria) [inline data_streaming.yml:46]:
```suggestion
          description: Too Many Requests
```

File: data_streaming.yml [hunk 2]
```diff
@@ -28,14 +27,152 @@ paths:
...
```
Comment (rodrigomaria) [inline data_streaming.yml:125]:
```suggestion
          description: Too Many Requests
```

File: data_streaming.yml [hunk 3]
```diff
@@ -28,14 +27,152 @@ paths:
...
```
Comment (rodrigomaria) [inline data_streaming.yml:152]:
```suggestion
          description: Too Many Requests
```

**Review by rodrigomaria — DISMISSED** [review]
LGTM

File: data_streaming.yml [hunk 4]
```diff
@@ -28,14 +27,152 @@ paths:
...
+        '500':
+          description: Internal Server Error
+  
+  /data_streaming/streamings/{data_streaming_id}:
+    get:
+      tags:
+        - Data Streaming
+      summary: Get expecific data streaming by Streaming ID
+      description: Return a data streaming info by ID
```
Comment (ftoldato) [inline data_streaming.yml:101]: description: Return a Data Streaming info by ID

**Review by ftoldato — APPROVED** [review]

**Review by rodrigomaria — APPROVED** [review]

**Pull request merged**